package.json

```diff
@@ -1035,6 +1035,17 @@
                         "type": "boolean",
                         "description": "%azureFunctions.endOfLifeWarning%",
                         "default": true
+                    },
+                    "azureFunctions.functionSubpath": {
```

Comment:
This one should be `"scope": "resource"` and the other one should not. Basically you want that if the setting can be specific to a folder/project and otherwise you don't need it.